docs/sources/auth/auth-proxy.md

```diff
@@ -25,6 +25,13 @@ enabled = true
 header_name = X-WEBAUTH-USER
 # HTTP Header property, defaults to `username` but can also be `email`
 header_property = username
+# Optionally define more headers to sync other user attributes
+# Example `headers = Name:X-WEBAUTH-NAME Email:X-WEBAUTH-EMAIL`
+headers =
+# If HTTP Header is a JSON string, set this to true. Defaults to 'false'.
+json_header = true
+# Needed when json_header is used, specifies the JSON key(s) for the user like username, email and/or full name.
+json_header_property = Login:preferred_username, Email:mail, Name:full_name
```

Comment:
```suggestion
json_properties = Login:preferred_username, Email:mail, Name:full_name
```

Reply:
Hi @DanCech, I would like to keep _header_ as a part of the name as then it is more clear that it goes together with json_header, so what would you say if I rename it to json_header_properties?

Reply:
I'm fine with that (and agree `json_header_properties` makes more sense), it just needs to be consistent everywhere, right now this is the only place that doesn't call it `json_properties`

Reply:
That's true, so I will use "json_header_properties" everywhere.